azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/service/calling/sdk/CallingSDKWrapper.kt

```diff
@@ -111,24 +113,32 @@ internal class CallingSDKWrapper(
 
     fun startCall(cameraState: CameraState, audioState: AudioState): CompletableFuture<Void> {
         val startCallCompletableFuture = CompletableFuture<Void>()
-        createCallAgent().thenAccept { agent: CallAgent ->
-            val audioOptions = AudioOptions()
-            audioOptions.isMuted = (audioState.operation != AudioOperationalStatus.ON)
-
-            var videoOptions: VideoOptions? = null
-            if (cameraState.operation == CameraOperationalStatus.ON) {
-                val localVideoStreams = arrayOf(getLocalVideoStream().get())
-                videoOptions = VideoOptions(localVideoStreams)
-            }
+        createCallAgent()
+            .completeOnTimeout(null, 15, TimeUnit.SECONDS)
+            .thenAccept { agent: CallAgent? ->
```

Comment:
@phileo99 is this workaround is for call join issue?

Reply:
I will revert the change